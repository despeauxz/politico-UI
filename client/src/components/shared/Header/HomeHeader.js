/* eslint-disable max-len */
import React, { Component } from 'react';
import { Link } from 'react-router-dom';
import Logo from '@base/img/logo2.png';
import Ham from '@base/img/ham.svg';
import Exit from '@base/img/exit.svg';


class HomeHeader extends Component {
    render() {
        return (
            <header className="home-landing">
                <Link to="/" className="">
                    <img src={Logo} alt="logo" className="logo" />
                </Link>

                <nav>
                    <a href="#" className="hide-desktop">
                        <img src={Ham} alt="toggle menu" className="menu" id="menu" />
                    </a>
                    <ul className="show-desktop hide-mobile" id="nav">
                        <li id="exit" className="exit-btn hide-desktop exit">
                            <img src={Exit} alt="Exit toggle" />
                        </li>
                        <li><Link to="/">Home</Link></li>
                        <li><Link to="#">Policy</Link></li>
                        <li><Link to="/auth/login">Login</Link></li>
                        <li><Link to="/auth/signup">Sign Up</Link></li>
                    </ul>
                </nav>
            </header>
        );
    }
}

export default HomeHeader;
