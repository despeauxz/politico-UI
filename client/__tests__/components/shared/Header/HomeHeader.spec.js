import React from 'react';
import HomeHeader from '@components/shared/Header/HomeHeader';
import toJson from 'enzyme-to-json';


describe('<HomeHeader />', () => {
    it('render the home header component', () => {
        const shallowWrapper = shallow(<HomeHeader />);

        expect(toJson(shallowWrapper)).toMatchSnapshot();
        expect(shallowWrapper.find('HomeHeader')).toBeTruthy();
    });
});
