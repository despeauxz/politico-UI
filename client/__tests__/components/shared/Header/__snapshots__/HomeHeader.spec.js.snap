// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeHeader /> render the home header component 1`] = `
<header
  className="home-landing"
>
  <Link
    className=""
    to="/"
  >
    <img
      alt="logo"
      className="logo"
      src={
        Object {
          "process": [Function],
        }
      }
    />
  </Link>
  <nav>
    <a
      className="hide-desktop"
      href="#"
    >
      <img
        alt="toggle menu"
        className="menu"
        id="menu"
        src={
          Object {
            "process": [Function],
          }
        }
      />
    </a>
    <ul
      className="show-desktop hide-mobile"
      id="nav"
    >
      <li
        className="exit-btn hide-desktop exit"
        id="exit"
      >
        <img
          alt="Exit toggle"
          src={
            Object {
              "process": [Function],
            }
          }
        />
      </li>
      <li>
        <Link
          to="/"
        >
          Home
        </Link>
      </li>
      <li>
        <Link
          to="#"
        >
          Policy
        </Link>
      </li>
      <li>
        <Link
          to="/auth/login"
        >
          Login
        </Link>
      </li>
      <li>
        <Link
          to="/auth/signup"
        >
          Sign Up
        </Link>
      </li>
    </ul>
  </nav>
</header>
`;
